public/tendersmash.js

```diff
@@ -430,6 +430,11 @@ app.controller("mainController", function ($scope, $http, $sce, $q, $timeout, pr
     return promises;
   }
 
+  $scope.getOrderedLists = function() {
+    var sorted = _.sortBy($scope.lists, function(x) { return x.name; });
```

Comment:
The way it is right now works well, as all the `area/` ones come up after the names, which feels kinda nice. Not sure if they should be first or last though...